### PR TITLE
MVP implementation

### PR DESCRIPTION
**Pull request by abu-usama**
Added modules
- `Mslm::EmailVerify`
- `Mslm::OTP`
and:
- `Mslm` that wraps both the above modules.

Also added the release script.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.vscode/
+/Mslm/blib/
```
Comment (UmanShahzad) [inline .gitignore:2]: Think absolute path shouldnt be used? Just `Mslm/blib` or do we wanna only ignore a `blib` at the root

File: Mslm/Makefile.PL [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    NAME              => 'Mslm',
+    VERSION_FROM      => 'lib/Mslm.pm',
+    PREREQ_PM         => {
+        'LWP::UserAgent'          => '0',
+        'JSON'                    => '0',
+        'URI'                     => '0',
+        'Exporter'                => '0'
+    },
+    ABSTRACT_FROM     => 'lib/Mslm.pm',
+    AUTHOR            => 'Abu Usama <usamaliaqat5846@gmail.com>',
```
Comment (UmanShahzad) [inline Mslm/Makefile.PL:16]: Mslm email

File: Mslm/lib/Mslm/OTP.pm [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+
+=item * C<$otp_token_verify_req> (hash reference) - A hash reference containing the OTP token verification request parameters.
+
+=item * C<%opts> (hash) - Optional parameters. You can pass in the following opts: C<base_url>, C<user_agent>, C<api_key>, and C<http_client>. These options will only work for the current request.
+
+=back
+
+=head1 AUTHOR
+
+Abu Usama, C<< <usamaliaqat5846@gmail.com> >>
```
Comment (UmanShahzad) [inline Mslm/lib/Mslm/OTP.pm:296]: Email - search any other place pls

Comment (UmanShahzad) [reply Mslm/lib/Mslm/OTP.pm:296]: Also you can probably just use official Mslm email for all these locations and say 'Mslm' as the author name, to keep it generic

**Comment by UmanShahzad** [comment]
Code wise looks like in the right direction

File: Mslm/MANIFEST [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+MANIFEST
+README
+t/01-usage.t
+t/manifest.t
+t/pod-coverage.t
+t/pod.t
+lib/Mslm.pm
+lib/Mslm/Common.pm
+lib/Mslm/EmailVerify.pm
+lib/Mslm/OTP.pm
```
[the remark this answers is not in the document]
Comment (abu-usama) [reply Mslm/MANIFEST:12]: Each file must be named individually. And also, the release tests check if all the files within the package are listed in the MANIFEST.

**Review by UmanShahzad — APPROVED** [review]

**Pull request merged**